appium/webdriver/webdriver.py

```diff
@@ -691,6 +691,32 @@ def battery_info(self):
         """
         return self.execute_script('mobile: batteryInfo')
 
+    @property
+    def session_capabilities(self):
```

Comment:
how about having `session` name?

Reply:
yeah, I think it make more sense to call it session.